research/sentiment_analysis/data/imdb.py

```diff
@@ -0,0 +1,38 @@
+import tensorflow as tf
+import numpy as np
+from data.util import to_dataset, pad_sentence, START_CHAR, OOV_CHAR
+
+NUM_CLASS = 2
+
+
+def construct_input_fns(vocabulary_size, sentence_length, batch_size, repeat=1):
+  """Returns a tuple of input functions, one for training, the other for evaluation.
+  Args:
+    vocabulary_size: The number of the most frequent tokens to be used from the corpus.
+    sentence_length: The number of words in each sentence. Longer sentences get cut, shorter ones padded.
+    batch_size: Number of data in each batch.
+    repeat: The number of epoch.
+  Raises:
+    ValueError: if the dataset value is not valid.
+  """
+  (_x_train, _y_train), (_x_test, _y_test) = tf.keras.datasets.imdb.load_data(path="imdb.npz",
```

Comment:
The format is really breaking here. The length limit is 80. https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#32-line-length.

Please use some proper IDE with Google style template.